src/string_util.h

```diff
@@ -65,6 +65,35 @@ using std::stoul;  // NOLINT(misc-unused-using-decls)
 #endif
 // NOLINTEND
 
+/**
+ * Check whether the given value is a power of two or not.
+ * @param val the value to check
+ */
+bool IsPowerOfTwo(const int64_t& val);
```

Comment:
do these need to be exposed in the API?  i think only `FormatHumanReadable` is accessed by a different implementation file.

Reply:
Yes this method doesnt need to be exposed. Exposing Base10 and Base2 is needed for tests and may be needed by other functions in the future.